### ISSUE TITLE
Write "Accessibilty" into the Handbook

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/MoFo-Engineering-Handbook#33]
We should have better guidelines around Accessibility. Let's do it!
